filebeat/module/nats/log/_meta/fields.yml

```diff
@@ -0,0 +1,11 @@
+- name: pid
```

Comment:
I don't think you need these here. They are already defined in ECS on the top level. Defining these here would mean these are fields under nats.log.pid for example. So far it seems we don't have any NATS specific fields which is actually great as it maps well to ECS: https://github.com/elastic/ecs